_scripts/runDockerRun.sh

```diff
@@ -10,7 +10,7 @@ proxy=""
 
 if [ "${CI:-}" != "true" ]
 then
-	proxy="-v $GOPATH/pkg/mod/cache/download:/cache -e GOPROXY=file:///cache"
+	proxy="-v $(go env GOPATH)/pkg/mod/cache/download:/cache -e GOPROXY=file:///cache"
```

Comment:
Thanks - good catch. 

Although to be 100% correct I think you need just the first element:

```
go env GOPATH | sed -e 's/:/\n/' | head -n 1
```

Reply:
Ah, good point!